src/orquestra/quantum/operators/_pauli_operators.py

```diff
@@ -35,7 +35,7 @@
 
 import numpy as np
 
-from ..circuits import Circuit, Operation, builtin_gate_by_name
+from orquestra.quantum.circuits import Circuit, Operation, builtin_gate_by_name
```

Comment:
[question] Why removing relative import here?

Reply:
I think this is something my isort did automatically. I will fix. 🙂